packages/contracts/src/dollar/ERC1155SetUri/ERC1155BurnableSetUri.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.7.0) (token/ERC1155/extensions/ERC1155Burnable.sol)
+
+pragma solidity ^0.8.0;
```

Comment:
We recently upgrade all of them to 0.8.16; due to security linters but I hope that merging this in will preserve those changes.